src/table/cells/handle-key-press.js

```diff
@@ -54,42 +49,43 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-export const moveFocus = (evt, rootElement, rowIndex, colIndex, selState) => {
+export const moveFocus = (evt, rootElement, cellCoord, setFocusedCell, selState) => {
   preventDefaultBehavior(evt);
   removeCurrentFocus(evt);
   const rowAndColumnCount = getRowAndColumnCount(rootElement);
-  const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
-  moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
+  const nextCellCoord = arrowKeysNavigation(evt, rowAndColumnCount, cellCoord, selState);
+  focusCell(rowAndColumnCount.rowElements, nextCellCoord);
+  setFocusedCell(nextCellCoord);
```

Comment:
Could we make a choice to call them cell, like` focusedCell,` `setFocusedCell`, `nextCell`,  
or cellCoord, like `focusedCellCoord`, `setFocusedCellCoord`, `nextCellCoord` 
to keep those consistent? 
Personally, I prefer cell.

Reply:
Good point. I agree, since it is not the actual cell, but only the coordinates